main.go

```diff
@@ -136,7 +151,26 @@ func handleSquash(w http.ResponseWriter, issueComment IssueComment, git Git, git
 	return SuccessResponse{}
 }
 
-func handlePullRequest(w http.ResponseWriter, body []byte, git Git, githubClient *github.Client) Response {
+func handlePlusOne(w http.ResponseWriter, issueComment IssueComment, pullRequests PullRequests, repositories Repositories) Response {
```

Comment:
`handle` is a very generic prefix. Maybe ok if inherited from generic interface, but I think we can do better here. What does "handling" mean?